src/CMakeLists.txt

```diff
@@ -94,22 +95,20 @@ if (KERNEL_HEADERS_DIR)
   target_compile_definitions(runtime PUBLIC KERNEL_HEADERS_DIR="${KERNEL_HEADERS_DIR}")
 endif()
 
-execute_process(
-  COMMAND git describe --abbrev=4 --dirty --tags
-  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-  OUTPUT_VARIABLE BPFTRACE_VERSION
-  ERROR_VARIABLE GIT_DESCRIBE_ERROR
-  OUTPUT_STRIP_TRAILING_WHITESPACE
-  RESULT_VARIABLE retcode
+add_custom_command(
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/version.h
+  DEPENDS ${PROJECT_SOURCE_DIR}/.git/index
```

Comment:
Looks like nix does not like this trick:

```
$ nix build
error: builder for '/nix/store/6w9jk2s0gd9vf5ycyi8a81b2ia9s5x6k-bpftrace.drv' failed with exit code 1;
       last 10 log lines:
       >     CMAKE_EXPORT_NO_PACKAGE_REGISTRY
       >     CMAKE_POLICY_DEFAULT_CMP0025
       >
       >
       > -- Build files have been written to: /build/source/build
       > cmake: enabled parallel building
       > cmake: enabled parallel installing
       > building
       > build flags: -j32
       > ninja: error: '/build/source/.git/index', needed by 'src/version.h', missing and no known rule to make it
       For full logs, run 'nix log /nix/store/6w9jk2s0gd9vf5ycyi8a81b2ia9s5x6k-bpftrace.drv'.
```

Reply:
https://github.com/bpftrace/bpftrace/pull/3022

Reply:
Ah I see, will fix